feat(remote): advanced configurable CSRF token fetching for cds@7.5 (#556)
cap-js/docs: node.js/remote-services.md

    }
}
```

In this example, CSRF handling is enabled for the `API_BUSINESS_PARTNER` service, for regular requests (`csrf: true`) and requests made within batch operations (`csrfInBatch: true`).

#### Advanced Configuration

Actually `csrf: true` is a convenient preset. If needed, you can further customize the CSRF-token handling with additional parameters:

```json
"cds": {
    "requires": {
        "API_BUSINESS_PARTNER": {
            [...]
            "csrf": {  // [!code focus]
              "method": "get",  // [!code focus]
              "url": "..."  // [!code focus]
            }
        }
    }
}
```

Here, the CSRF-token handling is customized at a more granular level:

 - `method`: The HTTP method for fetching the CSRF token. The default is `head`.
 - `url`: The URL for fetching the CSRF token. The default is the resource path without parameters.

> Global configuration `cds.env.features.fetch_csrf = true` is deprecated.

Reviewer: ```suggestion
```

support for it is removed.

    }
}
```

In this example, CSRF handling is enabled for the `API_BUSINESS_PARTNER` service, for regular requests (`csrf: true`) and requests made within batch operations (`csrfInBatch: true`).

#### Advanced Configuration

> For more advanced scenarios, you can further customize the CSRF-token handling with additional parameters:

Reviewer 2: ```suggestion
Actually `csrf: true` is a convenient preset. If needed, you can further customize the CSRF-token handling with additional parameters:
```

```json
"cds": {
    "requires": {
        "API_BUSINESS_PARTNER": {

>             "kind": "odata",
>             "model": "srv/external/API_BUSINESS_PARTNER",

Reviewer 2: ```suggestion
            [...]
```

>             "csrf": {
>               "method": "get",
>               "url": "..."

Reviewer 2: ```suggestion
            "csrf": {  // [!code focus]
              "method": "get",  // [!code focus]
              "url": "..."  // [!code focus]
```

    }
}
```

In this example, CSRF handling is enabled for the `API_BUSINESS_PARTNER` service, for regular requests (`csrf: true`) and requests made within batch operations (`csrfInBatch: true`).

#### Advanced Configuration

For more advanced scenarios, you can further customize the CSRF-token handling with additional parameters:

```json
"cds": {
    "requires": {
        "API_BUSINESS_PARTNER": {
            "kind": "odata",
            "model": "srv/external/API_BUSINESS_PARTNER",
            "csrf": {
              "method": "get",
              "url": "..."

>             },
>             "csrfInBatch": {
>               "method": "get",
>               "url": "..."
>             }

Author: ```suggestion
            }
```
